cfgov/jinja2/v1/_includes/organisms/_vars-mega-menu.html

```diff
@@ -19,11 +19,34 @@
 #}
 
 {# TODO: Update file path for Featured Menu Content module when that is added. #}
-{% import 'templates/nav/about-us-media.html' as about_us_media %}
+{% import 'molecules/featured-menu-content.html' as featured_menu_content %}
 
 {# Featured Menu Content. #}
+{% set about_us_content = {
+    'link': {
+        'text': 'The CFPB: Working for you',
+        'url':  '/the-bureau/',
+    },
+    'image': {
+        'src': '/static/img/fmc-about-us-540x300.jpg',
+        'alt': 'Still from about the CFPB video.',
+    },
+    'body': 'This short video covers what the CFPB is and how we are working for American consumers.'
+} %}
+{% set poly_com_content = {
+    'link': {
+        'text': 'Resources to help you comply',
+        'url':  '/policy-compliance/guidance/implementation-guidance/',
+    },
+    'image': {
+        'src': '/static/img/fmc-poly-com-540x300.jpg',
+        'alt': 'Still from Know Before You Owe video.',
+    },
+    'body': 'The TILA-RESPA integrated disclosure rule replaces four disclosure forms with two new ones. We have resources to help you comply.'
```

Comment:
I would love for this to be in Django with and provide access like `getConfig('MegaMenu')`.